.github/actions/draft-release/src/index.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const { makeRelease } = require('./utils');
+const ReleaseChangelog = require('../../../../bin/release-changelog/changelog');
+
+async function main() {
+	try {
+		const milestone = core.getInput('milestone', { required: true });
+
+		// Do a sanity check on the milestone name.
+		if (!/^v[0-9]+\.[0-9]+(\.[0-9]+)?$/.test(milestone)) {
+			throw new Error('Milestone name is invalid.');
+		}
+
+		// Get tag name from plugin main PHP file.
+		let tagName = '';
+		const pluginFile = fs.readFileSync(path.resolve(process.cwd(), 'amp.php')).toString();
+		const matches = /\*\s+Version:\s+(\d+(\.\d+)+-\w+)/.exec(pluginFile);
+		if (matches && matches[1]) {
+			[, tagName] = matches;
+		}
+
```

Comment:
It might be worthwhile to do some checking of the milestone to make sure it matches:

```suggestion

	if ( ! tagName.startsWith( tagName.substr(1) ) ) {
		throw new Error( "Milestone mismatch with PHP plugin bootsrap version." );
	}

```

Reply:
Done in 752872daea14761701c6f2b7676f404c1aa9ec32.